project-base/docker/php-fpm/Dockerfile

```diff
@@ -57,6 +57,8 @@ RUN apk add --update --no-cache --repository http://dl-cdn.alpinelinux.org/alpin
 ENV LD_PRELOAD /usr/lib/preloadable_libiconv.so php
 ENV LC_ALL=en_US.utf8 LANG=en_US.utf8 LANGUAGE=en_US.utf8
 
+COPY docker-php-entrypoint /usr/local/bin/
```

Comment:
This overwrites the original docker-php-entrypoint (https://github.com/docker-library/php/blob/master/docker-php-entrypoint)

An inline comment would be helpful...